apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -206,6 +206,8 @@ export const WorkoutEditor = ({
         </InputGroup>
       </FormControl>
 
+      <Text fontWeight={'light'}>Tip:Use 0 as watt target for Free Mode</Text>
```

Comment:
Can call it free mode for now, but I can't really see _Free mode__ being used as a term. Some mention _Resistance mode_, _Standard mode_ and _SIM mode_ – we should decide what to call this 🤠 

Anyways, I have some minor suggestions for the text itself, and I also suggest moving the text beneath the _Add part_ button.

```tsx
      <Text fontSize="small">
        Tip: Use 0 as power target for <em>Free mode</em>
      </Text>
```

aka:
- Use `fontSize` instead of `fontWeight`
- We don't need `{}` around component String params
- Add spacing after `Tip:`
- Call it _Power target_ instead of _Watt target_, since there is a column named _Power_ above said field
- _Free mode_ instead of _Free Mode_ – That's also what's been used the other places it is mentioned in this PR 🤓 
- Add emphasis around `Free mode`